server/routes/visitJourney/dateAndTime.ts

```diff
@@ -15,17 +16,31 @@ export default class DateAndTime {
   ) {}
 
   async get(req: Request, res: Response): Promise<void> {
+    let daysUntilVisitStart = 2
+
     const isUpdate = this.mode === 'update'
     const { prisonId } = req.session.selectedEstablishment
     const { visitSessionData } = req.session
+
+    let warningBannerText = ''
```

Comment:
This change will mean that we might potentially have more than one warning message within the same warning component. I don't think this scenario is specified on the ticket, so need to check with Ed/Phoebe what to do in this case. Seems like here all messages will be in one paragraph.

Alternatives might be:
* bulleted individual messages within a single warning component
* multiple warning components; each with a single message

To avoid duplication in the template, an array of warning message strings could be passed, so it could be looped to render either of the above suggestions.